twiml/voice/gather/gather-1/output/gather-1.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Response>
-    <Gather input="speech dtmf" timeout="3" numDigits="1">
+    <Gather input="speech dtmf" numDigits="1" timeout="3">

```

Comment:
Do not reorder XML attributes for the test, this must not be sufficient for the test to fail. (See comment for using an XML parser instead of a regex).